src/sqlfluff/dialects/dialect_postgres_keywords.py

```diff
@@ -478,7 +478,7 @@ def get_keywords(keyword_list, keyword_type):
     ("MAXVALUE", "non-reserved"),
     ("MEASURES", "not-keyword"),
     ("MEMBER", "not-keyword"),
-    ("MERGE", "not-keyword"),
+    ("MERGE", "reserved"),
```

Comment:
I think this should be non-reserved:

```suggestion
    ("MERGE", "non-reserved"),
```

Also you need to update `MATCHED` to be a non-reserved keyword too.

Reply:
Do you see the second point here @PhilippLange ?

> Also you need to update MATCHED to be a non-reserved keyword too.